File: inventory-api/src/main/java/com/kruger/ec/dto/core/KgrtVaccinationDto.java
package com.kruger.ec.dto.core;


import com.kruger.ec.dto.AbstractDto;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import java.util.List;

@Getter
@Setter
public class KgrtVaccinationDto extends AbstractDto<Long> {

    private Long vaccinationlId;
    
    private Integer vaccinationiDose;
    
    private String vaccinationvObservation;
    
    private Date vaccinationdDate;
    
    private String vaccinationvPathLicense;
    
    private Boolean vaccinationbActive;
    
    private Date audidCreationDate;
    
    private String audivUserCreation;
    
    private Date audidModificationDate;
    
    private String audivUserModification;
    
    private Date audidDeleteDate;
    
    private String audivUserDelete;
    private KgrtEmployeeDto employeevDniDTO;
    private KgrtVaccineDto vaccinelIdDTO;
    private List<KgrtVaccinationStatusDto> kgrtVaccinationStatusCollectionDTO;

}

File: inventory-ui/src/app/dto/KgrtVaccinationDto.ts
import {KgrtEmployeeDto} from "@/dto/KgrtEmployeeDto";
import {KrgtVaccineDto} from "@/dto/KgrtVaccineDto";
import {KgrtVaccinationStatusDto} from "@/dto/KgrtVaccinationStatusDto";

export class KgrtVaccinationDto {
  vaccinationlId: number;
  vaccinationiDose: number;
  vaccinationvObservation: string;
  vaccinationdDate: Date;
  vaccinationvPathLicense: string;
  vaccinationbActive: boolean;
  audidCreationDate: Date;
  audivUserCreation: string;
  audidModificationDate: Date;
  audivUserModification: string;
  audidDeleteDate: Date;
  audivUserDelete: string;
  employeevDniDTO: KgrtEmployeeDto;
  vaccinelIdDTO: KrgtVaccineDto;
  kgrtVaccinationStatusCollectionDTO: KgrtVaccinationStatusDto[];
}
